src/Bicep.LangServer/Snippets/Templates/res-firewall-standard-vwan.bicep

```diff
@@ -0,0 +1,22 @@
+// Azure Firewall Standard SKU for Virtual WAN Hub
+resource /*${1:firewall}*/firewall 'Microsoft.Network/azureFirewalls@2021-05-01' = {
+  name: /*${2:'name'}*/'name'
+  location: /*${3:location}*/'location'
+  properties: {
+    sku: {
+      name: 'AZFW_Hub'
+      tier: 'Standard'
+    }    
+    firewallPolicy: {
+      id: /*${4:'firewallPolicy.id'}*/'firewallPolicy.id'

```

Comment:
Shouldn't be in quotes, e.g.:

id: /*${4:firewallPolicy.id}*/'firewallPolicy.id

Reply:
We have references like this in many snippets, wouldn't pass the tests otherwise I guess. I can make the change but that would require to add the firewallPolicy to the snippet using the `existing` keyword, and I'm not sure if we want that?

Reply:
You can specify the values that will be replaced for the tabstops in the tests using these comments:

// $1 = firewall
// $2 = 'name'
// $3 = location
// $4 = 'firewallPolicy.id'
// $5 = 'virtualHub.id'
// $6 = 2

As long as those are compilable (which they are above), I think it should pass the tests.

Reply:
And yes, it would be good to standardize those better.  My preference though going forward is to suggest the actual code (even if it doesn't compile) rather than a string literal that compiles but doesn't make sense.

@ucheNkadiCode - I'd rather see this in the snippet (as a tab stop):

id: virtualHub.id

than this:

id: 'virtualHub.id'

Let me know if you disagree thx.